hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/mapreduce/MapReduceRestoreToOriginalSplitsJob.java

```diff
@@ -45,7 +45,9 @@ public void run(Path[] dirPaths, TableName[] fromTables, Path restoreRootDir,
     // We are using the files from the snapshot. We should copy them rather than move them over
     conf.setBoolean(BulkLoadHFiles.ALWAYS_COPY_FILES, true);
 
-    FileSystem fs = FileSystem.get(conf);
+    Path backupRootDir = new Path(conf.get(RestoreJob.BACKUP_ROOT_PATH_KEY));
+
+    FileSystem fs = backupRootDir.getFileSystem(conf);
```

Comment:
This looks good to me. How does the typical `MapReduceRestoreJob` handle this? Wondering if we should knock out a similar issue there while we're here

Reply:
the `MapReduceRestoreJob` doesn't have this issue. It delegates the files out to a MR job which re-writes the splits, then bulkloads from the output. The MR must be smart enough to pull from the correct file system. 

as a safety measure, I've extended the test I added to restore using new splits from a remote FS, and confirmed that it works